### DISCUSSION TITLE
SQL Custom Field Plugin: Set header attribute option

### DISCUSSION THREAD

**Discussion by TheWhykiki** [joomla/joomla-cms#30607] [2. Cold Features]
### Steps to reproduce the issue

Add custom field type SQL. There is no possibility to set a header option to the dropdown field. So I'm also not able to leave the value of the field empty if needed.

### Expected result

A new field type Text in der Fields XML, maybe named "Header" where we can leave a String or maybe also a JText String.
If field is filled a header attribute is set for the SQL-Field, if empty there will be no header and the first result from the SQL-Quer will be shown.

### Actual result

No header field, no good possibility to leave the value of this field empty or set a default value for start.

### System information (as much as possible)

![screen shot 2020-09-09 at 19 34 32](https://issues.joomla.org/uploads/1/4bf69b5ad090996824bd536e542a97c6.png)
### Additional comments

It would only need a few lines of code to add this feature in the PHP and XML

![screen shot 2020-09-09 at 19 34 32](https://issues.joomla.org/uploads/1/f6eafb536b2e1bebb32e04201eccd221.png)

**Comment by SharkyKZ** [comment]
Until someone decides to implement this you can use UNION to select an empty value.
